### PR TITLE
Updated help messages

### PR DESCRIPTION
**Pull request by brunats**
**- What I did**

Updated help messages for core commands and commands with more than one subcommand.
_This PR includes PR https://github.com/ZupIT/ritchie-cli/pull/256 changes for the ritchie-2.0.0 branch_

> Core
antes
->
depois

>Completion
completion1
> ->
completion2

> Delete
delete1
->
aa

>Set
set1
->
set2

**- How to verify it**

```bash
rit # to verify help messages of commands core
rit completion
rit delete
rit set
```

**- Description for the changelog**

Updated help messages

### CODE REVIEW

File: functional/core/core_feature.json [hunk 1]
```diff
@@ -224,7 +224,7 @@
                 "action":"main"
             }
         ],
-        "result":"Delete objects like users, etc."
+        "result":"Delete a object like context and repository."
```
Comment (sandokandias) [inline functional/core/core_feature.json:227]: Use the same pattern of `set` description

File: functional/core/core_feature.json [hunk 2]
```diff
@@ -63,7 +63,7 @@
                 "action":"main"
             }
         ],
-        "result":"It can be used to add formulas, repositories and other objects.."
+        "result":"Add a new formula repository to ritchie-cli."
```
Comment (sandokandias) [inline functional/core/core_feature.json:66]: "Add a new repository of formulas"

File: pkg/cmd/add.go [hunk 3]
```diff
@@ -4,17 +4,12 @@ import (
...
-
 // NewAddCmd create a new add instance
 func NewAddCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "add SUBCOMMAND",
-		Short: "add objects",
-		Long:  descAddLong,
+		Use:     "add SUBCOMMAND",
+		Short:   "Add repositories ",
+		Long:    "Add a new formula repository to ritchie-cli.",
```
Comment (sandokandias) [inline pkg/cmd/add.go:12]: "Add a new repository of formulas"

File: pkg/cmd/build.go [hunk 4]
```diff
@@ -4,8 +4,9 @@ import "github.com/spf13/cobra"
 
 func NewBuildCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "build SUB_COMMAND",
-		Short: "This is a root command, needs a sub command",
-		Long:  `This is a root command, to use it, add a sub command. For example, rit build formula.`,
+		Use:     "build SUB_COMMAND",
+		Short:   "Build formulas - with subcomando formulas",
```
Comment (sandokandias) [inline pkg/cmd/build.go:8]: "Build formulas"

File: pkg/cmd/build.go [hunk 5]
```diff
@@ -4,8 +4,9 @@ import "github.com/spf13/cobra"
...
+		Long:    "This is a root command, to use it, add a sub command.",
```
Comment (sandokandias) [inline pkg/cmd/build.go:9]: Use the same pattern of the add description.

File: pkg/cmd/delete.go [hunk 6]
```diff
@@ -5,8 +5,9 @@ import "github.com/spf13/cobra"
 // NewDeleteCmd create a new delete instance
 func NewDeleteCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "delete SUBCOMMAND",
-		Short: "Delete objects",
-		Long:  `Delete objects like users, etc.`,
+		Use:     "delete SUBCOMMAND",
+		Short:   "Delete objects (contexts, repositories)",
```
Comment (sandokandias) [inline pkg/cmd/delete.go:9]: Use the same pattern of set description

File: pkg/cmd/delete.go [hunk 7]
```diff
@@ -5,8 +5,9 @@ import "github.com/spf13/cobra"
...
+		Long:    "Delete a object like context and repository.",
```
Comment (sandokandias) [inline pkg/cmd/delete.go:10]: Use the same pattern of set description

File: pkg/cmd/delete_context.go [hunk 8]
```diff
@@ -31,9 +31,9 @@ func NewDeleteContextCmd(
 
 	cmd := &cobra.Command{
 		Use:     "context",
-		Short:   "Delete context for Ritchie-cli",
+		Short:   "Delete context for ritchie-cli",
```
Comment (sandokandias) [inline pkg/cmd/delete_context.go:34]: We need to be more clear, `ritchie-cli` is the program, so maybe you can use `delete context for credentials`

File: pkg/cmd/set.go [hunk 9]
```diff
@@ -7,8 +7,9 @@ import (
 // NewSetCmd creates new cmd instance
 func NewSetCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "set SUBCOMMAND",
-		Short: "Set objects",
-		Long:  `Set objects like credentials, etc.`,
+		Use:     "set SUBCOMMAND",
+		Short:   "Set objects (context, credential, repo-priority)",
```
Comment (sandokandias) [inline pkg/cmd/set.go:11]: "Set contexts, credentials and priorities

File: pkg/cmd/add.go (hunk 3 above)
Comment (brunats) [reply pkg/cmd/add.go]: https://github.com/ZupIT/ritchie-cli/pull/311/commits/7e0e7eea21f2e7412f2e8451cd04284f72dba1c0

File: functional/core/core_feature.json (hunk 2 above)
Comment (brunats) [reply functional/core/core_feature.json]: Fix https://github.com/ZupIT/ritchie-cli/pull/311/commits/7e0e7eea21f2e7412f2e8451cd04284f72dba1c0

File: functional/core/core_feature.json (hunk 1 above)
Comment (brunats) [reply functional/core/core_feature.json]: Fix in https://github.com/ZupIT/ritchie-cli/pull/311/commits/7e0e7eea21f2e7412f2e8451cd04284f72dba1c0

File: pkg/cmd/build.go (hunk 4 above)
Comment (brunats) [reply pkg/cmd/build.go]: Fix in https://github.com/ZupIT/ritchie-cli/pull/311/commits/7e0e7eea21f2e7412f2e8451cd04284f72dba1c0

File: pkg/cmd/build.go (hunk 5 above)
Comment (brunats) [reply pkg/cmd/build.go]: Fix in https://github.com/ZupIT/ritchie-cli/pull/311/commits/04b4392aa18003a6637cfb23196fcd9460bb48a5

File: pkg/cmd/delete.go (hunk 6 above)
Comment (brunats) [reply pkg/cmd/delete.go]: Fix in https://github.com/ZupIT/ritchie-cli/pull/311/commits/7e0e7eea21f2e7412f2e8451cd04284f72dba1c0

File: pkg/cmd/delete.go (hunk 7 above)
Comment (brunats) [reply pkg/cmd/delete.go]: Fix in https://github.com/ZupIT/ritchie-cli/pull/311/commits/7e0e7eea21f2e7412f2e8451cd04284f72dba1c0

File: pkg/cmd/set.go (hunk 9 above)
Comment (brunats) [reply pkg/cmd/set.go]: Fix https://github.com/ZupIT/ritchie-cli/pull/311/commits/7e0e7eea21f2e7412f2e8451cd04284f72dba1c0

**Pull request merged**